RELEASE-NOTES.rst

```diff
@@ -13,23 +13,30 @@ dictionaries.
 New features
 ------------
 
-- Adds tolerance parameter used when user wants to treat closed values
-  as equals
+-- Adds support for comparing NumPy arrays.  (#68)
+-- Adds support for comparing mutable mappings, sequences and sets from
+-  `collections.abs` module.  (#67)
 
 Improved features
 -----------------
 
-- Adds support for comparing numerical values and NaN. (#54) (#55)
```

Comment:
@jirikuncar The `kwalitee` tool gave only one dash, but I added two to be consistent with the previous version.